tools/fast/series/mining_once_from_ctx.go

```diff
@@ -0,0 +1,19 @@
+package series
+
+import (
+	"context"
+)
+
+// CKMiningOnce is the key used to store the MiningOnceFunc in a context
+var CKMiningOnce = struct{}{}
+
+// MiningOnceFunc is the type for the value stored under the CKMiningOnce key
+type MiningOnceFunc func()
+
+// MiningOnceFromCtx extracts the MiningOnceFunc through key CKMiningOnce
+func MiningOnceFromCtx(ctx context.Context) {
+	MiningOnce, ok := ctx.Value(CKMiningOnce).(MiningOnceFunc)
+	if ok {
+		MiningOnce()
+	}
+}
```

Comment:
This is a solution to enabling the ability to run `MiningOnce` in integration tests.

Integration tests in general do not setup with async mining (`mining start`). The issue with using async mining is the lack of control over block height, and the increased test time that can come from it. One of the goals of FAST is to enable reuse of code. A simple example of this is sending tokens from one node to another. Ideally, in a single function call, the sending of the message, and the waiting would be performed to ensure that when the function returns, the message has been mined and the targeted wallet has the funds. This works well when there is async mining because the function can wait around for the message. However, when working with tests that need to have more control over the block height, eg. payment channel expiry, using async miners requires more padding around expiry values and waiting around for blocks to be mined. This increases the test time substantially. 

The idea is that this can be used in FAST series where mining may be required, such as sending tokens. When running in a functional testing environment (devnets with async mining) the operation is a noop. When explicit mining control is required, a MiningOnceFunc can be added to the context enabling series to manage explicit mining without having to pass in a dedicated miner node, or requiring that all nodes be miners (would suck).